packages/client/src/generators.ts

```diff
@@ -0,0 +1,37 @@
+import { fromByteArray } from 'base64-js';
+import { sha256 } from 'js-sha256';
+
+function generateRandomData(length: number) {
```

Comment:
we could have an open issue on this one to discuss if there is a better or more elegant approach.